src/clients/twitch.ts

```diff
@@ -1,9 +1,12 @@
+import bridge, { genericPromiseError, configFile, manageMsgCache } from './bridge';
```

Comment:
configFile might be the only one you need to destructure here, according to the search tool, `genericPromiseError` and `manageMsgCache` only come up with two results, so they could use the original `bridge` import ^_^

Reply:
The Bridge import is just the `bridge` object as defined in.. bridge.ts, which is an object which does not contain `genericPromiseError`, or `manageMsgCache`. The `bridge` import is just the default import from bridge!
We could add `genericPromiseError` or `manageMsgCache` to the `bridge` default export, although that'd be pretty yikes.

Reply:
Ah gotcha XP